pkg/subnet/subnet.go

```diff
@@ -15,6 +17,84 @@ type ipAndNetwork struct {
 // stubbable version
 var interfaceAddrs = net.InterfaceAddrs
 
+// IPv4 subnet key. Identifies a class B subnet
+type ipv4SubnetKey [2]byte
+
+// IPv6 subnet key. This is the 48-bit route and 16-bit subnet identifier. Identifies a 64 bit subnet.
+type ipv6SubnetKey [8]byte
```

Comment:
Style suggestion:

Because these don't have any methods, and are only used inside of `AnalyzeIPs()`, go ahead and define the types inside of the `AnalyzeIPs()` body.